src/plugins/discover/server/saved_objects/search.ts

```diff
@@ -47,5 +47,5 @@ export const searchSavedObjectType: SavedObjectsType = {
       version: { type: 'integer' },
     },
   },
-  migrations: searchMigrations as any,
+  migrations: searchMigrations,
```

Comment:
how come this works now?

Reply:
I don't know what broke that earlier actually, but it's perfectly fine typed for this use-case :)